### ISSUE TITLE
Не совсем понятно, как работают команды

### ISSUE THREAD

**Issue by ladanovn** [fletcherist/yandex-dialogs-sdk#90]
````
const Alice = require('yandex-dialogs-sdk')
const alice = new Alice()

alice.welcome(async ctx => ctx.reply('welcome'));
alice.command(['Показать мои устройства', 
               'Показать ваши устройства',
               'Что у меня есть ?'], async ctx => ctx.reply('сморти'));
alice.command('Включи свет в ${where}', async ctx => {
  ctx.reply(`Включил свет в ${ctx.body.where}`);
});
      
alice.any((ctx) => {
  ctx.reply('Увы, я пока не знакома с этой командой')
});

alice.listen('/', 8000);
````
![screen](https://image.ibb.co/eo4dy9/2018_08_10_18_09_06.png)

**Comment by fletcherist** [comment]
@ladanovn hi there! It seems to be a bug :(
That would be fixed in 2.0. Sorry for that unpredictable stuff.

At that moment you can do smth like that to fix the problem

```javascript
alice.command(ctx => {
return ['Показать мои устройства', 
  'Показать ваши устройства',
  'Что у меня есть ?'].includes(ctx.message)
}, async ctx => ctx.reply('сморти'));
```

**Comment by fletcherist** [comment]
@ladanovn Fixed. You can check out new **README** 👍
